### ISSUE TITLE
EOsimilar_strings_generate_similar_hashesTest.testWorks -> StackOverflow

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/eo#1383]
I've faced with the next exception during packaging `eo-maven-plugin`:
```
EOsimilar_strings_generate_similar_hashesTest.testWorks » StackOverflow
```
The problem happens not often. In other words, it's a _flushing_ test that happened several times on my own laptop and once during CI Pipeline: [here is it](https://github.com/objectionary/eo/actions/runs/3360717185/jobs/5570249481#step:5:3695).
To reproduce just run `mvn clean install` several times.
The full exception log:
```java
[ERROR] EOorg.EOeolang.EOsimilar_strings_generate_similar_hashesTest.testWorks  Time elapsed: 1.146 s  <<< ERROR!
java.lang.StackOverflowError
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
        at org.eolang.AtNamed.put(AtNamed.java:112)
...
```

**Comment by yegor256** [comment]
@volodya-lombrozo we need to increase stack size for surefire plugin, similar to how we did it here: https://github.com/objectionary/eoc/pull/89

**Comment by volodya-lombrozo** [comment]
@yegor256 again https://github.com/objectionary/eo/actions/runs/3379047008/jobs/5610088030

**Comment by volodya-lombrozo** [comment]
https://github.com/objectionary/eo/actions/runs/3408728269/jobs/5669649433

**Comment by volodya-lombrozo** [comment]
@yegor256 we have to set `-Xss` param for CI pipelines to decrease the number of failed checks

**Comment by ivan-egorov42** [comment]
@volodya-lombrozo It will be available soon: https://github.com/objectionary/eo/pull/1444

**Comment by yegor256** [comment]
@volodya-lombrozo this is fixed in #1448
